includes/class-genesis-simple-faq-widget.php

```diff
@@ -101,7 +107,7 @@ public function widget( $args, $instance ) {
 				);
 
 				// Allow filtering of the template markup.
-				echo apply_filters( 'gs_faq_template', $template, $question, $answer );
+				echo wp_kses_post( apply_filters( 'gs_faq_template', $template, $question, $answer ) );
```

Comment:
Possible that this could filter out something that someone has added to the template. Thoughts @nickcernis?

Reply:
I agree that this could potentially strip HTML from user templates. We should consider omitting wp_kses_post here with a phpcs ignore comment.

Reply:
Should be fixed by the last commit @dreamwhisper @nickcernis